docs/index.md

```diff
@@ -0,0 +1,5 @@
+# MkDocs
```

Comment:
Instead of adding Lorem Ipsum, it's generally better practice to leave documentation that isn't (or can't be) completed blank, so that it isn't possible for this non-documentation content to sneak into a final release.